tests/client/test_encryption_client.py

```diff
@@ -0,0 +1,37 @@
+import os
+import unittest
+
+from qingstor.sdk.client.encryption_client import EncryptionClient
+
+ZONE="test_zone"
+ENCRYPT_KEY=b"0"*32
+CONFIG="test_config"
+CLIENT="test_client"
+BUCKET_NAME="test_bucket"
+IN_KWAGRS={u"content_type":u"video"}
+RIGHT_OUT_KWARGS={
+    u"content_type":u"video",
```

Comment:
use `from __future__ import unicode_literals` instead of add `u` for every unicode.